### PR TITLE
cargo: sync workspace members with dependencies

### PR DESCRIPTION
**Pull request by FlorianFranzen**
This syncs the workspace crates returned by `cargo metadata` with the crates inside `Cargo.toml`, which resolves some issues I was having with naersk as well with some older lsp servers.

### CODE REVIEW

**Review by bkchr — APPROVED** [review]

File: Cargo.toml [hunk 1]
```diff
@@ -90,9 +96,24 @@ members = [
...
+	"bridges/modules/grandpa",
+	"bridges/modules/messages",
+	"bridges/primitives/chain-rococo",
+	"bridges/primitives/chain-wococo",
+	"bridges/primitives/header-chain",
+	"bridges/primitives/message-dispatch",
+	"bridges/primitives/messages",
+	"bridges/primitives/polkadot-core",
+	"bridges/primitives/runtime",
+	"bridges/primitives/test-utils",
```
Comment (ordian) [inline Cargo.toml:116]: I'm not sure about bridges pallets as it's added as a subtree as was removed here last time by @HCastano 
cc @svyatonik @acatangiu

**Review by ordian — APPROVED** [review]
Needs a merge with master and looks good other than a question about bridges.

File: Cargo.toml (hunk 1 above)
Comment (bkchr) [reply Cargo.toml:116]: Yeah I already discussed this with @FlorianFranzen, but he still doesn't has removed them..

Comment (svyatonik) [reply Cargo.toml:116]: Imo there's no harm in adding these crates to workspace - they're built with the code anyways. We should avoid adding all `bridges/*` crates - there are crates that shall not be built here (ideally we should remove them from the polkadot repo, but ...).

**Comment by FlorianFranzen** [comment]
@bkchr Thank you, not sure why this dropped off my notifications.

**Pull request merged**